### PR TITLE
DOC-3949: Added subsection on ephemeral ports

### PR DESCRIPTION
**Pull request by tonyjhillman**

### CODE REVIEW

**Comment by tonyjhillman** [comment]
Approval by David pending.

File: modules/install/pages/install-ports.adoc [hunk 1]
```diff
@@ -9,14 +9,22 @@ These ports must be open for Couchbase Server to operate correctly.
...
-IMPORTANT: If other software on the same host is using any of the ports that are required by Couchbase Server, then Couchbase Server will not function properly and may fail to start. 
+IMPORTANT: If other software on the same host is using any of the ports that are required by Couchbase Server, then Couchbase Server will not function properly and may fail to start.
+
+=== Ephemeral Ports
+
+An _ephemeral_ port is one temporarily allocated by a server's operating system, to handle client-communications initiated on a _well-known_ port that is currently busy.
+For Couchbase Server, an example of a well-known port would be _8091_.
+
+Each operating system provides a default range of port numbers that can be assigned to ephemeral ports, when necessary.
+Couchbase Server relies on the _full_ default range provided by each operating system: therefore, the default range should _not_ be reduced by the administrator; since reduction may cause Couchbase-Server processes to fail.
```
Comment (amarantha-k) [inline modules/install/pages/install-ports.adoc]: "Couchbase-Server processes" -> "Couchbase Server processes"

**Review by amarantha-k — APPROVED** [review]
lgtm

**Comment by tonyjhillman** [comment]
Will open another PR for this.

**Pull request closed without merging**